### PR TITLE
Adding Newton per Minute units

### PR DESCRIPTION
**Pull request by salido-m**

### CODE REVIEW

File: UnitsNet/Scripts/UnitDefinitions/ForceChangeRate.json [hunk 1]
```diff
@@ -3,6 +3,19 @@
   "BaseUnit": "NewtonPerSecond",
   "XmlDoc": "Force change rate is the ratio of the force change to the time during which the change occurred (value of force changes per unit time).",
   "Units": [
+{
+      "SingularName": "NewtonPerMinute",
+      "PluralName": "NewtonsPerMinute",
+      "FromUnitToBaseFunc": "x*60",
+      "FromBaseToUnitFunc": "x/60",
```
Comment (anjdreas) [inline UnitsNet/Scripts/UnitDefinitions/ForceChangeRate.json]: I think the two functions should be swapped?

**Comment by salido-m** [comment]
Yes, you're right, i swapped the the two functions, and updated tests values.

File: UnitsNet/Scripts/UnitDefinitions/ForceChangeRate.json (hunk 1 above)
Comment (salido-m) [inline UnitsNet/Scripts/UnitDefinitions/ForceChangeRate.json]: Yes, you're right, i swapped the two functions, and updated tests values.

**Comment by anjdreas** [comment]
Awesome, thanks

**Comment by anjdreas** [comment]
Nuget 3.48 out.

**Pull request merged**